yarn.lock

```diff
@@ -2424,6 +2424,11 @@
   version "0.0.0"
   uid ""
 
+"@lwc/shared@2.22.0":
```

Comment:
Something is wrong here. The `@lwc/shared` lib should be resolved locally.

Reply:
Removed it.